uTinyRipperCore/Structure/ProjectCollection/Collections/SceneExportCollection.cs

```diff
@@ -220,10 +220,14 @@ private string GetSceneName(IExportContainer container)
 			{
 				int index = FileNameToSceneIndex(Name, File.Version);
 				string scenePath = container.SceneIndexToName(index);
-				int assetsNameIndex = scenePath.IndexOf(AssetsName, StringComparison.Ordinal);
-				if (assetsNameIndex != -1)
+				bool rooted = DirectoryUtils.IsPathRooted(AssetsName, out int prefixLength);
+				if (rooted || assetsNameIndex == 0)
 				{
-					string relativePath = scenePath.Substring(assetsNameIndex + AssetsName.Length);
+					if (rooted && assetsNameIndex != -1)
+					{
+						prefixLength = assetsNameIndex;
+					}
+					string relativePath = scenePath.Substring(prefixLength + AssetsName.Length);
```

Comment:
if assetsNameIndex equals to -1, don't need to add AssetsName.Length
btw, assetsNameIndex  should never be equals to -1, since it is part of editor project